packer.py

```diff
@@ -15,6 +16,7 @@ def load_box_types(file="boxes.json"):
         return 1
```

Comment:
Just noticed this. In many languages `True` converts to `1`, and vice-versa, so better not to return 1 if something fails.
In this case you must ask self if exception here is recoverable. Can we proceed without loaded file `boxes.json` ? I think no, so exception guard is not needed here. Just let exception break execution in this case.

Reply:
Agree. Will change it in next commit.